corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -0,0 +1,127 @@
+
+        # let user chose a project from the list ['commcare-mobile-2.21', ...]
+        selected_project_key = None
+        print "Jenkins has following projects. Please choose one (to end enter END)"
+        print [p for p in self.jenkin_projects if 'commcare-mobile' in p]
+        while selected_project_key not in self.jenkin_projects:
+            selected_project_key = raw_input("")
+            if selected_project_key.lower() == 'end':
+                return
+
+        print "Fetching build information for %s. Please wait..." % selected_project_key
+        selected_project = self.build_server[selected_project_key]
+        build_dict = selected_project.get_build_dict()
+        builds_by_version_number = self._extract_version_numbers(build_dict)
+
+        if not builds_by_version_number:
+            abort = raw_input("This project doesn't have any builds that has a VERSION set. Do you want"
+                              " to chose another project  (Yes) or abort (No).")
+            if abort.lower() == 'no':
+                print "Builds URL http://jenkins.dimagi.com/job/", selected_project_key
+                return
+            else:
+                return self.interactive_import()
+            return
+
+        selected_build_number = None
+        print "Jenkins has following builds for %s. Choose a build-number to import (to end enter 0)" \
+              % selected_project_key
+        print builds_by_version_number
+        while selected_build_number not in builds_by_version_number.keys():
+            if selected_build_number == 0:
+                return
+            selected_build_number = int(raw_input(""))
+
+        # download and add the build
+        print "Downloading and importing artifacts.zip. Please wait..."
+        version_number = builds_by_version_number.get(selected_build_number)
+        build = selected_project.get_build_metadata(selected_build_number)
+        artifacts = build.get_artifact_dict()
+        artifacts_url = artifacts['artifacts.zip'].url
+
+        try:
+            zip_file = requests.get(artifacts_url)
+        except:
+            print "Failed to fetch artifacts.zip at URL`"
+            print artifacts_url
+            return
+
+        self.add_build(StringIO(zip_file.content), version_number, selected_build_number)
+
+    @property
+    @memoized
+    def jenkin_projects(self):
+        print "Pinging Jenkins build server. Pelase wait..."
+        return self.build_server.keys()
+
+    @property
+    @memoized
+    def build_server(self):
+        return Jenkins("http://jenkins.dimagi.com")
+
+    def add_build(self, builf_file, version, build_number):
+        try:
+            CommCareBuild.create_from_zip(builf_file, version, build_number)
+        except Exception as e:
+            raise CommandError("%s" % e)
+        self.stdout.write('Build %s #%s created\n' % (version, build_number))
+        self.stdout.write('You can see a list of builds at [your-server]/builds/')
```

Comment:
You can get the whole URL using `corehq.utils.view_utils.absolute_reverse` (although it looks like you'd need to name the corresponding URLs).  Alternatively, `get_url_base()` will do the trick.